app/workers/flickr_photos_importer.rb

```diff
@@ -62,13 +62,16 @@ def store_photo(flickr_photo_structure, group_id)
     FlickrPhoto.create(attributes, op_type: 'create')
   rescue Elasticsearch::Transport::Transport::Errors::Conflict
     script = {
-      source: 'ctx._source.popularity = new_popularity',
-      lang: 'groovy',
+      source: 'ctx._source.popularity = params.new_popularity ;',
```

Comment:
why a space before semicolon?

Reply:
I was following our original style, but it looks like the standard is no space, so I'll remove it.